dts/xtensa/intel_s1000.dtsi

```diff
@@ -31,7 +31,7 @@
 	sram0: memory@be000000 {
 		device_type = "memory";
 		compatible = "mmio-sram";
-		reg = <0xbe000000 0x400000>;
+		reg = <0xbe030000 0x3D0000>;
```

Comment:
This change affects all applications built for S1000 and the flash image generation script as well.
There should be a better way to do this - perhaps with **2 different DTS files**?

- mcuboot should be built to run from a lower (or upper) portion memory
- other applications should be built to run from remaining portion of the memory.

**Using the upper portion of memory for mcuboot might allow of easier switch between applications built pre-mcuboot-era and applications built post-mcuboot-era**

Reply:
As discussed offline, have done it via the overlays.